src/layouts/VideoGrid.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from "react"
```

Comment:
VideoGrid seems more of a component than a layout.
If it is not being used as a layout please place it in components folder.